README.markdown

```diff
@@ -22,3 +22,51 @@ long the outcome is distributed under the same license. Otherwise, you must
 obtain a [commercial license](./LICENSE-COMMERCIAL) that removes such
 restrictions. Read more about our licensing approach
 [here](https://www.sourcemeta.com/licensing/).
+
+Example
+--------------
+
+```cpp
+#include <sourcemeta/jsontoolkit/json.h>
+#include <sourcemeta/jsontoolkit/jsonpointer.h>
+#include <sstream>
+#include <iostream>
+
+int main()
+{
+    // creating json using parse
```

Comment:
You should be careful with comments:

- Casing is important. `creating json` reads poorly, while `Creating JSON` is proper casing
- Semantics and terminology are important. You don't "create json" using parse. You "parse a JSON document" instead